drivers/counter/counter_gecko_rtcc.c

```diff
@@ -172,7 +172,7 @@ static int counter_gecko_set_alarm(struct device *dev, u8_t chan_id,
 		return -EBUSY;
 	}
 
-	if (alarm_cfg->absolute) {
+	if (alarm_cfg->flags & COUNTER_ALARM_CFG_ABSOLUTE) {
```

Comment:
Misra-C pendantry: need to write this as `((alarm_cfg->flags & COUNTER_ALARM_CFG_ABSOLUTE) != 0)`, here and elsewhere

Reply:
done for all drivers.